python/mxnet/gluon/data/dataloader.py

```diff
@@ -277,12 +298,16 @@ def default_batchify_fn(data):
 
     num_workers : int, default 0
         The number of multiprocessing workers to use for data preprocessing.
-        `num_workers > 0` is not supported on Windows yet.
+    pin_memory : boolean, default False
```

Comment:
Why default to False?

Reply:
not necessary for non-gpu instances, and using non-pagable memory for everything can screw up the stability